### PR TITLE
brew-script: using composite

### PR DESCRIPTION
**Pull request by dawidd6**

### CODE REVIEW

File: brew-script/action.yml [hunk 1]
```diff
@@ -13,5 +13,19 @@ inputs:
...
+      shell: bash
+    - run: brew ruby /tmp/brew-script-$GITHUB_SHA
+      shell: bash
+      env:
+        HOMEBREW_GITHUB_API_TOKEN: ${{inputs.token}}
+        HOMEBREW_NO_ENV_FILTERING: 1
+        HOMEBREW_NO_AUTO_UPDATE: 1
+        HOMEBREW_NO_ANALYTICS: 1
+        HOMEBREW_COLOR: 1
+        RUBYOPT: -W0
```
Comment (MikeMcQuaid) [inline brew-script/action.yml:31]: Why set this?

File: brew-script/action.yml [hunk 2]
```diff
@@ -13,5 +13,19 @@ inputs:
...
+        ${{inputs.script}}
+        EOF
+      shell: bash
+    - run: brew ruby /tmp/brew-script-$GITHUB_SHA
+      shell: bash
+      env:
+        HOMEBREW_GITHUB_API_TOKEN: ${{inputs.token}}
+        HOMEBREW_NO_ENV_FILTERING: 1
+        HOMEBREW_NO_AUTO_UPDATE: 1
+        HOMEBREW_NO_ANALYTICS: 1
```
Comment (MikeMcQuaid) [inline brew-script/action.yml:29]: Why set this?

File: brew-script/action.yml (hunk 1 above)
Comment (dawidd6) [reply brew-script/action.yml:31]: So one does not see messages like this:
```
/home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/language/python.rb:8: warning: Insecure world writable dir /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/shims/scm in PATH, mode 040777

```

File: brew-script/action.yml (hunk 2 above)
Comment (dawidd6) [reply brew-script/action.yml:29]: Could remove it if you think it isn't needed. Just thought we don't want analytics from this Action.

File: brew-script/action.yml (hunk 1 above)
Comment (MikeMcQuaid) [reply brew-script/action.yml:31]: We should fix that message either in Homebrew or, ideally, in the environment we execute this in.

File: brew-script/action.yml (hunk 2 above)
Comment (MikeMcQuaid) [reply brew-script/action.yml:29]: Yeh, we don't want to disable analytics anywhere really.

Comment (dawidd6) [reply brew-script/action.yml:29]: https://github.com/Homebrew/actions/commit/faee221d871037213f8d0e936fb113246b003539

File: brew-script/action.yml (hunk 1 above)
Comment (dawidd6) [reply brew-script/action.yml:31]: https://github.com/Homebrew/brew/pull/8407

Deleted in: https://github.com/Homebrew/actions/commit/f4b98344cb593dc08583251f91d6701719a0c873

File: brew-script/action.yml (hunk 2 above)
Comment (MikeMcQuaid) [reply brew-script/action.yml:29]: Thanks on both!

**Pull request merged**